core/src/test/scala/org/apache/spark/storage/ShuffleBlockFetcherIteratorSuite.scala

```diff
@@ -1219,7 +1219,12 @@ class ShuffleBlockFetcherIteratorSuite extends SparkFunSuite with PrivateMethodT
             new RuntimeException("forced error"))
         }
```

Comment:
Also update the other tests, particularly related to push based shuffle to assert on the expected metrics ?
Ideally, the original tests should have done it (if the metrics had been added initially) - and so now we have to retroactively fix them ...

Reply:
We can do this as a follow up, given the size of this PR.
Can you please file a jira for this ? Thanks !